.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+JIRA Link: <link or N/A>
+
```

Comment:
MAYBE have a section that gives details for why the change is happening:

i.e. 

```
Issues Addressed:

Summary of Changes:
```